SwiftUIDZ/SwiftUIDZ/VerificationView.swift

```diff
@@ -0,0 +1,240 @@
+// VerificationView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран верификации
+struct VerificationView: View {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let yellowColorName = "pastelYellow"
+        static let purpleColorName = "pastelPurple"
+        static let darkPurpleColorName = "pastelDarkPurple"
+        static let navBarHeight: CGFloat = 10
+        static let oneHundredFiftyFontSize: CGFloat = 150
+        static let twoFloat: CGFloat = 2
+        static let twentyPadding: CGFloat = 20
+        static let seventyPadding: CGFloat = 70
+        static let navigationTitle = "Verification"
+        static let backButtonIconName = "chevron.left"
+        static let oneHundredPadding: CGFloat = 100
+        static let envelopeIconName = "envelope.fill"
+        static let verificationCodeText = "Verification code"
+        static let thirtyPadding: CGFloat = 30
+        static let cornerRadius: CGFloat = 8
+        static let continueButtonTitle = "Continue"
+        static let checkSmsText = "Check the sms"
+        static let messageText = "message to get a verification code"
+        static let receiveSmsText = "Did’t receive sms"
+        static let sendSmsAgainButtonTitle = "Send sms again"
+        static let tenPadding: CGFloat = 10
+        static let fillFromMessage = "Fill in from message"
+        static let okButtonTitle = "Ok"
+        static let actionSheetInfoText = "We already know about the bug and fix it soon"
+        static let totalValueProgress = 100.0
+    }
+
+    // MARK: - Private enum
+
+    private enum FocusedField {
+        case first, second, third, fourth
+    }
+
+    // MARK: - Body
+
+    var body: some View {
+        VStack {
+            GradientNavigationView(navBarHeight: Constants.navBarHeight)
+            envelopeImageView
+            verificationCodeTextView
+            codeTextFields
+            checkSmsTextView
+            continueButtonView
+            sendSmsAgainView
+                .padding(.top, Constants.twentyPadding)
```

Comment:
```suggestion
                .padding(.top, Constants.twentyPadding)
```
убери эту строчку в sendSmsAgainView